solidity/contracts/OwnableMulticall.sol

```diff
@@ -32,23 +33,28 @@ contract OwnableMulticall is OwnableUpgradeable {
         }
     }
 
-    function _call(Call[] memory calls, bytes[] memory callbacks)
+    function _staticcall(Call[] memory calls, bytes[] memory callbacks)
         internal
-        returns (bytes[] memory resolveCalls)
+        view
+        returns (Result[] memory resolveCalls)
     {
-        resolveCalls = new bytes[](callbacks.length);
-        for (uint256 i = 0; i < calls.length; i++) {
-            (bool success, bytes memory returnData) = calls[i].to.call(
+        uint256 length = calls.length;
+        resolveCalls = new Result[](length);
+        for (uint256 i = 0; i < length; i++) {
+            (bool success, bytes memory returnData) = calls[i].to.staticcall(
                 calls[i].data
             );
-            require(success, "Multicall: call failed");
-            resolveCalls[i] = bytes.concat(callbacks[i], returnData);
+            resolveCalls[i] = Result(
+                success,
+                bytes.concat(callbacks[i], returnData)
```

Comment:
Feels kind of weird to concat if success == False, no?

Reply:
I guess we don't do anything with that if success==false, in which case maybe it's an opportunity for gas savings?